python/mxnet/contrib/onnx/mx2onnx/_op_translations.py

```diff
@@ -127,6 +127,14 @@ def convert_string_to_list(string_val):
 
     return result_list
 
+def convert_bool_to_int(attrs, attr_name):
```

Comment:
you are not converting boolean values to int. you converting string values to int. I think this should be named differently, not sure what though

Reply:
you can call it `get_attribute_value`

Reply:
`get_boolean_attribute_value`? because this helper gets only bool valued attributes

Reply:
ok